README.md

```diff
@@ -1,5 +1,40 @@
 # SourceCred Research
 
-This repository is used to keep track of SourceCred research matters.
+This repository contains research infrastructure, code, and results for SourceCred.
 
-We will soon add a description of the attribution algorithm, and open research problems.
+## Setup
+
+Our research codebase is written in Python3, and makes use of Jupyter notebooks.
+We recommend using [Conda] to create and manage Python environments. See the
```

Comment:
Interesting—curious why you recommend Conda over vanilla virtualenv+pip?

I’m not too familiar with Conda, so this question may be ill-founded,
but: is it generally “safe” to mix `conda install` with `pip install`?
I’ve seen bizarre issues with shared library compatibility when mixing
Conda and Pip that don’t occur with Pip alone; e.g.:
<https://github.com/tensorflow/tensorboard/issues/1999>

It’s also just a higher barrier to entry. I’ve been using Pip for many
years, but the first time that I installed Conda ever was last week.

Reply:
I suggested conda because it's Jupyter "strongly recommend"s using Conda.

https://jupyter.org/install

I don't feel super strongly about it myself. Although, using just `pip` I managed to f*ck up my system python pretty badly (i forget how) and my personal fix trajectory passed through a state where my OS was incapable of rendering GUIs, and I needed to fix it on the TTY. (To be clear: that was my fault.) But this convinced me that just lazily using `pip` to install at the system level and hope things work is, in fact, a bad idea.

It's probably fine for people to use virtualenv or whatever, though. Maybe we should rephrase the README to only "suggest" conda.